sdk/spring-sdk-testkit/src/main/java/kalix/springsdk/testkit/EventSourcedTestKit.java

```diff
@@ -22,9 +22,11 @@
 import kalix.javasdk.testkit.EventSourcedResult;
 import kalix.javasdk.testkit.impl.EventSourcedEntityEffectsRunner;
 import kalix.javasdk.testkit.impl.TestKitEventSourcedEntityContext;
+import kalix.springsdk.impl.SpringSdkMessageCodec;
 import kalix.springsdk.impl.eventsourcedentity.EventSourceEntityHandlers;
 import kalix.springsdk.impl.eventsourcedentity.EventSourcedHandlersExtractor;
 
+import javax.swing.*;
```

Comment:
Almost Friday, but still earlier to start swinging.
Good try! 😂

Reply:
good catch! not sure what happened here 🤔 😂